### ISSUE TITLE
wget: TLS error from peer (alert code 80): 80

### ISSUE THREAD

**Issue by Marko-TRG** [aserto-dev/topaz#359]
These messages appear in the minikube log when attempting 'Deploy as a Sidecar':

```
get-eds-citadel Connecting to github.com (20.87.245.0:443)
get-eds-citadel wget: TLS error from peer (alert code 80): 80
get-eds-citadel wget: error getting response: Connection reset by peer
```

The problem is the old version of wget, because of an older image of busybox (1.28) in deployment.yaml. 
Changing this to 1.36.1 (latest as of writing) solved the problem:

`
image: busybox:1.36.1
`

**Comment by ronenh** [comment]
Thanks for reporting the issue @Marko-TRG.
The problem was fixed last week in https://github.com/aserto-dev/topaz/pull/357 and released in topaz v0.31.10.

**Comment by Marko-TRG** [comment]
Thanks, Ronen!
